apps/squareone/sentry.edge.config.js

```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry for edge features (middleware, edge routes, and so on).
```

Comment:
I don't think we actually need this, but I'm very shaky on how NextJS actually works :zany_face: